source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,80 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
+ - Pre-packaged Calls version [v0.25.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.25.0).
+ - Pre-packaged GitHub plugin version [v2.2.0](https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.2.0).
+ - Upgraded ``@mattermost/client`` and ``@mattermost/types`` to support TypeScript v5.x.
+ - The first emoji is now auto-selected in the emoji picker.
+ - Added markdown support for batched email notifications.
+ - Users’ timezone is now used in batched email notifications.
+ - Removed a conflicting class (``help-text``) from the interactive dialog field description to resolve the black text color in the dark theme.
+ - Updated the user interface of **Team Settings** modal.
+ - Promoted Simplified Chinese to Beta, and downgraded Hungarian and Spanish languages to Beta.
+ - Improved the opening animation of the user settings modal.
+
+#### Administration
+ - Dropped pre-packaged plugins for unsupported OS and architectures.
+ - Implemented a new **Export Settings** page in the **System Console** to allow Cloud administrators to customize their dedicated export S3 buckets.
+ - LDAP job details are no longer shown until the job runs.
+ - Added more logging to the ``NotificationsLog``.
+ - A message is now logged if a user tries to login using a wrong password.
+ - Posts from deactivated users are now included in **Direct Message** channel exports. Also the ``--include-archived-channels`` flag is now respected for **Direct Message** channels.
+ - Changed the cache headers for file endpoints to cache privately for 24 hours, instead of not caching at all.
+ - Moved following functions from server to public utils:
+    - ``ResetReadTimeout``
+    - ``AppendMultipleStatementsFlag``
+    - ``SetupConnection``
+    - ``SanitizeDataSource``
+
+#### mmctl
+ - mmctl can now be used to download a Support Packet using ``--local mode``.
+ - mmctl system ping will now return detailed server status even if the server status is unhealthy.
+
+### Bug Fixes
+ - Fixed an issue where the Desktop App login flow would erroneously show the landing page for first time users.
+ - Fixed an issue where a right-hand side card was not reloaded when the card body was updated.
+ - Fixed an issue where ``en-AU`` language selection was not allowed.
+ - Fixed an issue with the position of text in the default profile picture.
+ - Fixed an issue with the group search to parse the display name.
+ - Fixed an issue where items with longer text did not widen the user guide dropdown to its max-width.
+ - Fixed an issue where the configuration could not be updated from the **System Console** in cloud environments.
+
+### config.json
+ - Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
```

Comment:
>  Below is a list of the additions and their default values on install

Where is this list?

Reply:
Below it on line 71 https://github.com/mattermost/docs/pull/6985/files#diff-0c47e93e01b6bf186c28619073df3eb12f83259e7181ea91fffa1c48575d2296R71.